src/System.Security.Cryptography.Xml/src/Resources/SR.Designer.cs

```diff
@@ -0,0 +1,838 @@
+﻿//------------------------------------------------------------------------------
```

Comment:
Hmm - perhaps I got confused before, now when I removed the item and renamed to Strings.resx, it all just works. Never mind :)